### PR TITLE
Refine marginal definition

### PR DESCRIPTION
**Pull request by Henry-WattTime**

### CODE REVIEW

File: Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ Market-based measures are financial instruments designed to neutralize or offset
   <tbody>
     <tr>
 	<td><strong>Marginal Carbon Intensity (I) </strong></td>
-	<td>This is the emissions intensity of the marginal power plant which will be turned up if you schedule some compute (e.g. increase electricity demand from the grid) at that moment.</td>
+	<td>This is the change in emissions caused by a change in electricty consumption. That is the emissions intensity of the marginal power plant which will be turned up if you schedule some compute (e.g. increase electricity demand from the grid) at that moment.</td>
```
Comment (atg-abhishek) [inline Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md:282]:
```suggestion
	<td>This is the change in emissions caused by a unit change in electricity consumption. That is the emissions intensity of the marginal power plant which will be turned up if you schedule some compute (e.g. increase electricity demand from the grid) at that moment.</td>
```

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
LGTM, minor change made to spelling and addition of the word "unit"

**Review by atg-abhishek — CHANGES_REQUESTED** [review]
LGTM, minor change made to spelling and addition of the word "unit"

**Comment by Henry-WattTime** [comment]
WG: This is no longer relevant as this section will be removed. Closing without changing.

**Pull request closed without merging**